### ISSUE TITLE
Still maintened?

### ISSUE THREAD

**Issue by VascoSch92** [KieranWynn/pyquaternion#85]
Is this repository/package still maintened?

**Comment by RohitDhankar** [comment]
Yes +1 that query -- also is there anyone who still uses any of this -- like to rephrase -- have you guys heard of Neural Networks Deep Learning ? is any of this still relevant ?
